### DISCUSSION TITLE
刮削成功，但是除了封面以外都沒有应用

### DISCUSSION THREAD

**Discussion by hstonc** [metatube-community/jellyfin-plugin-metatube#345] [Q&A]
本地后台部署
./metatube-server-linux-amd64 -port 9000 -dsn metatube.db
后台的日志输出显示刮削是成功，但是除了封面和影视数据库编号以外都沒有应用
![2024-06-13 21-05-12 的螢幕擷圖](https://github.com/metatube-community/jellyfin-plugin-metatube/assets/23431385/676cf90b-d287-4894-b64f-55091be827bc)
还需要设置是东西吗

**Comment by YuleLee-652987** [comment]
它上头说了选了就不能更改数据了，你全取消试试。

**Reply by hstonc** [reply]
> 它上头说了选了就不能更改数据了，你全取消试试。

反选才是锁定，勾选是不锁定，两种都试过了，还是一样
